srcs/httpMethod/HTTPProcess.cpp

```diff
@@ -15,7 +15,7 @@
  * @author ymori
  *
  */
-std::string http_process(ft::ServerChild server_child) {
+std::string http_process(ft::ServerChild& server_child) {
```

Comment:
I made it a reference so we can update serverChild status code. 
we can close the connection by making the status code >= 400